### PR TITLE
Validation form

### PR DESCRIPTION
**Pull request by AbbasSarwar**
### Simple validation
* Form validation Css & Js
* Email inputs only small
* Text error shows when the email is not valid

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,10 +118,13 @@ In order to run this project you need:
...
-- Twitter: [@TambanaEssohana](https://twitter.com/TambanaEssohana)
-- LinkedIn: [@TambanaEssohana](https://www.linkedin.com/in/essohanam-tambana-62aa081a1/?originalSubdomain=tg)
+- GitHub: [@sheldonwafula](https://github.com/sheldonwafula)
 
+👤 **Author3**
+
+- GitHub: [@Abmart0803](https://github.com/Abmart0803)
+- Twitter: [@MartinsAbmart](https://twitter.com/MartinsAbmart)
+- LinkedIn: [@abmartcodingworld](https://www.linkedin.com/in/abmartcodingworld/)
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (wayungi) [inline README.md]: - [ ] Your project history shows 2 contributors but your README has 3 authors. Kindly fix this discrepancy.
authors

File: js/validation.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            event.preventDefault()
+        }else if(email.match(/[@]/g) == false){
+            error.innerHTML ='@ missing'
+            event.preventDefault()
+        } else {
+            error.innerHTML = ''
+            return true
+        }
+    }
+    checkEmail()
```
Comment (wayungi) [inline js/validation.js]: - [OPTIONAL] I think it would be better if you defined the `checkEmail()` separately and not inside the event listener

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @AbbasSarwar , 

Good job working with your partner on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Your ESLint is not running. Kindly set it up to run.
- [ ] Your email case validation is failing. No error is displayed around the submit button or form fields when I submit an capiltalised email address.
emial

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/validation.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+let form = document.getElementById('form')
+let email = document.getElementById('mail')
+let error = document.querySelector('.error')
+form.addEventListener('submit', (event)=>{
+    event.preventDefault()
```
Comment (didierganthier) [inline js/validation.js]: Well done overall 💯 but the preventDefault() method of the [Event](https://developer.mozilla.org/en-US/docs/Web/API/Event) interface tells the [user agent](https://developer.mozilla.org/en-US/docs/Glossary/User_agent) that if the event does not get explicitly handled ❌, its default action should not be taken as it normally would be. Therefore this line is causing your `form` to not get submitted (the default behavior) even when it's error-free, removing it makes everything works perfectly. 😄

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi Team,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.



## Highlights
- Descriptive PR ✔️ 
- Validation check ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- [ ] Well done overall 💯 however the linters aren't configured quite well yet, you're using the `HTML + CSS` when you should've been using the [HTML + CSS + JS Linters](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) considering that your project contains `JS` code. This is an invalidation reason but since it was not done by the previous reviewer I just required the changes 😅  so I strongly encourage you to update your `linters.yml` file with that one [here](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) to ensure that you have all the linters check. ✅ 

- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by Munsa1 — APPROVED** [review]
Hi @AbbasSarwar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Implemented all required chanes :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**